www/nginx/src/opnsense/mvc/app/models/OPNsense/Nginx/Nginx.xml

```diff
@@ -205,6 +205,9 @@
         <Required>N</Required>
         <multiple>N</multiple>
       </upstream>
+      <new_urlpattern type="TextField">
+        <Required>N</Required>
```

Comment:
needs a mask regex pattern - usually no whitespace allowed which is `\S+`, maybe you also want to limit the length to something useful by using {,100} for a maximum length of 100 instead of +.